docs/operate/get-started/setup-micro/_index.md

```diff
@@ -0,0 +1,404 @@
+
+- [`gpio`](/operate/reference/components/servo/gpio-micro-rdk/): A servo controlled by GPIO pins.
+- [`two_wheeled_base`](/operate/reference/components/base/two_wheeled_base/): A robotic base with differential steering.
+- [`free_heap_sensor`](https://github.com/viamrobotics/micro-rdk/tree/main/examples/modular-drivers/src): Reports the amount of free heap memory on the microcontroller.
+- [`wifi_rssi_sensor`](https://github.com/viamrobotics/micro-rdk/tree/main/examples/modular-drivers/src): Reports the signal strength of the ESP32's WiFi connection.
+
+## Quickstart with pre-built firmware
+
+{{% hiddencontent %}}
+Viam provides installers to flash an ESP32 from macOS or Linux running on the x86_64 or ARM64 (AArch64) processor architectures.
+{{% /hiddencontent %}}
+
+To get started quickly with the pre-built `viam-micro-server` binary, follow these steps:
+
+1. Create a [Viam app](https://app.viam.com) account.
+   The Viam app is the online hub for configuring and managing devices as well as viewing data.
+
+1. Add a new _{{< glossary_tooltip term_id="machine" text="machine" >}}_ using the button in the top right corner of the **LOCATIONS** tab in the app.
+   A machine represents your device.
+
+1. From your machine's page in the Viam app, click **View setup instructions** and follow the steps for your operating system.
+   The app provides commands to install `viam-micro-server` and connect it to the cloud with your machine's unique credentials.
+
+1. A secure connection is automatically established between your machine and the Viam app.
+   When you update your machine's configuration, `viam-micro-server` automatically gets the updates.
+
+   You are ready to [configure](/operate/get-started/supported-hardware/) any of the components listed above on your machine.
+
+## Build and flash custom firmware
+
+If you are using your ESP32 with resources that are not supported by the pre-built binary, you can build your own firmware with the Micro-RDK and your choice of {{< glossary_tooltip term_id="module" text="modules" >}}.
+
+### Set up your development environment
+
+The [Micro-RDK](https://github.com/viamrobotics/micro-rdk) (from which `viam-micro-server` is built) is written in Rust.
+To be able to develop for the Micro-RDK on macOS and Linux systems, you must install the following software on your development machine:
+
+1. Install dependencies:
+
+   {{< tabs >}}
+   {{% tab name="Linux" %}}
+
+   ```sh { class="command-line" data-prompt="$"}
+   sudo apt-get install bison ccache cmake curl dfu-util flex git gperf libffi-dev libssl-dev libudev-dev libusb-1.0-0 ninja-build python3 python3-pip python3-venv wget
+   ```
+
+   {{% /tab %}}
+   {{% tab name="macOS" %}}
+
+   If you haven't already, install Homebrew:
+
+   ```sh { class="command-line" data-prompt="$"}
+   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+   ```
```

Comment:
```suggestion
   If you haven't already, [install Homebrew](https://brew.sh/):
```
[opt] Homebrew's installation instructions, which basically boil down to 'download and immediately run a bash script and give it admin privileges quickly thereafter' has always sketched me out a little. If they ever change, updating our own 'install homebrew' instructions will be kind of annoying. Because of this, I prefer to point users to the Homebrew site to follow the instructions from there, instead.

Reply:
Okay sure! Removing colon
```suggestion
If you haven't already, [install Homebrew](https://brew.sh/).
```